web/modules/custom/asset_distribution/asset_distribution.behat.inc

```diff
@@ -167,37 +165,17 @@ class AssetDistributionSubContext extends DrupalSubContextBase implements Drupal
       if (array_key_exists($key, $aliases)) {
         $values[$aliases[$key]] = $value;
       }
-      // If the 'parent' key is present we will have to create a relation to the
-      // parent entity, which is either a release or a solution.
-      elseif ($key === 'parent') {
-        $parent_entity = self::getRdfEntityByLabel($value);
-
-        // If we know the parent then we can transparently populate the group
-        // reference to the solution.
-        if (!array_key_exists('solution', $aliased_values)) {
-          $solution_label = NULL;
-          if ($parent_entity->bundle() === 'solution') {
-            $solution_label = $parent_entity->label();
-          }
-          elseif ($solution_entity_id = $parent_entity->field_isr_is_version_of->value) {
```

Comment:
I found the core of the relation issues. It is the `->value` part of this line. It will never retrieve the value as the `field_isr_is_version_of` is an entity reference, and the magic method to retrieve the value is `->target_id`... We have been having this in our code for a while. It has given us a false positive in some tests. I think it is a very good thing to have finally gotten rid of it. 
Your relations work fine up to now.